model_types/modelTypesList.json

```diff
@@ -18,12 +18,12 @@
       "modelCategory": "synthetics",
       "defaultConfig": "config_templates/gretel/synthetics/natural-language.yml",
       "sampleDataset": {
-        "fileName": "taylor-swift-lyrics-up.csv",
+        "fileName": "banking-77-6class.csv",
         "description": "Create believable song lyrics using this dataset of Taylor Swift hits.",
```

Comment:
You'll want to update this description, too. :)

Reply:
Nice catch. Thanks!